scripts/build.ps1

```diff
@@ -323,6 +323,8 @@ function Create-VsixPackage
     $vsixProjectDir = Join-Path $env:TP_OUT_DIR "$TPB_Configuration\VSIX"
     $packageDir = Get-FullCLRPackageDirectory
     $testhostPackageDir = Join-Path $packageDir "TestHost"
+    $extensionsPackageDir = Join-Path $packageDir "Extensions"
+    $testImpactComComponentsDir = Join-Path $extensionsPackageDir "TestImpact"
```

Comment:
This is getting consumed by test impact collector. The way collector is implemented today requires us to put com components in Extensions/TestImpact directory